.github/workflows/build.yaml

```
@@ -206,14 +206,15 @@ jobs:
           logout: false
       - name: push
         run: |
+          apt-get update && apt-get install -y wget
+          wget https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq
```

```suggestion
          wget https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq
```

There is a `yq` action if we wanted to use it (https://mikefarah.gitbook.io/yq/usage/github-action), but I'm fine with the one-liner.